skopt/searchcv.py

```diff
@@ -319,6 +291,11 @@ def _check_search_space(self, search_space):
         if isinstance(search_space, dict):
             search_space = [search_space]
 
+        if len(self.search_spaces) == 0:
+            raise ValueError(
+                "Please provide at least one search space"
```

Comment:
-> `"The search_spaces parameter has to contain at least one search space not %s" % search_space` I find it helps the user (aka me debugging at 3am) a lot to have the parameter name and the current value of it in the message.

Reply:
thank you for providing the code 😸